cedar-drt/fuzz/fuzz_targets/partial-eval.rs

```diff
@@ -78,8 +68,16 @@ impl<'a> Arbitrary<'a> for FuzzTargetInput {
     fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
         let schema = Schema::arbitrary(SETTINGS.clone(), u)?;
         let hierarchy = schema.arbitrary_hierarchy(u)?;
-        let policy = schema.arbitrary_policy(&hierarchy, u)?;
-
+        let toplevel_type = arbitrary_schematype_with_bounded_depth(
+            &SETTINGS,
+            schema.entity_types(),
+            SETTINGS.max_depth,
+            u,
+        )?;
+
+        let expr_gen = schema.exprgenerator(Some(&hierarchy));
+        let expression =
+            expr_gen.generate_expr_for_schematype(&toplevel_type, SETTINGS.max_depth, u)?;
```

Comment:
will this generate expressions that include unknowns?

Reply:
yes